### DISCUSSION TITLE
Use blocks editor instead of traditional editor

### DISCUSSION THREAD

**Discussion by hugocm99** [jeffreyvr/easy-liveblogs#9] [Ideas]
I have the doubt if it will be possible in some way to improve the Easy Liveblog plugin using in its entries editor Wordpress blocks instead of the classic editor, as it gives more options and things than the classic editor?

**Comment by jeffreyvr** [comment]
Good idea. At some point in the future we probably need to make the transition anyway once the classic editor is no longer supported by WordPress.

**Reply by hugocm99** [reply]
Yes, that’s true, the tendency of Wordpress is going to the block editor (on widgets pages for example they changed not so long ago) and it will bring the possibility for example to embed twitter or YouTube content in an easier way than with the classic editor.

I have been searching for a little bit of info but I don’t really know if it’s to difficult or it’s easy to make the change in this.
